map/bookmark.hpp

```diff
@@ -96,7 +97,13 @@ class Bookmark : public ICustomDrawable
   void SetName(string const & name) { m_data.SetName(name); }
   /// @return Now its a bookmark color - name of icon file
   string const & GetType() const { return m_data.GetType(); }
-  void SetType(string const & type) { m_data.SetType(type); }
+
+  void SetType(string const & type)
+  {
+    m_data.SetType(type);
+    SetStyle(type);
```

Comment:
StyledPoint::m_style duplicates BookmarkData::m_type. Why do we need this duplication? Is it possible to get rid of StyledPoint::m_style and introduce ApiMarkPoint::m_style instead?